doc/manual.rst

```diff
@@ -3994,7 +3990,7 @@ the rewriting step, so that all overloads of ``items``/``pairs`` are taken
 into account.
 
 
-First class iterators
+First-class iterators
```

Comment:
If you build the entire documentation with `koch docs` and then view this page, you'll see that this change causes the URL anchor to be `#iterators-and-the-for-statement-firstminusclass-iterators` (this can be seen when you hover over the header). In particular, note that spaces are replaced with `-` but an actual `-` is replaced with `minus` instead. The anchor also includes any parent headers.

Thus, the link back here: https://github.com/nim-lang/Nim/pull/15747/files#diff-8ae43d01f18469b14179d32881322ffe36482c28633d86f8941750d45827f7b9R907
...is broken since it still tries to link to `#iterators-and-the-for-statement-first-class-iterators`.

I'm not entirely sure of the finer details, but some special characters (such as the slash, and of course, the minus sign) are replaced with their "names", for lack of a better term, in URL anchors.

Yeah, this isn't obvious. I only know this since I did submit some documentation-related PRs quite a while ago :)

Reply:
Thanks a lot. Hopefully, it has been fixed...